runtime-modules/proposals/engine/src/types/mod.rs

```diff
@@ -379,15 +379,12 @@ pub struct ProposalCreationParameters<BlockNumber, Balance, MemberId, AccountId>
     pub exact_execution_block: Option<BlockNumber>,
 }
 
-/// Contains staking balance and account id.
+/// Contains staking account id.
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, Default, Clone, Copy, PartialEq, Eq, Debug)]
-pub struct Stake<AccountId, Balance> {
+pub struct Stake<AccountId> {
```

Comment:
:D Does this struct really make sense at this point! you are desperate not to go back to just a simple account? :P

Reply:
Ok. I removed it.